cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinUnit.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.cristin.person.model.cristin;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import nva.commons.core.JacocoGenerated;
+
+@SuppressWarnings("unused")
+@JacocoGenerated
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
```

Comment:
Why do we need a special naming strategy?

Reply:
Data from Cristin use underscore case